cloud-init/web.txt

```diff
@@ -34,7 +34,7 @@ write_files:
         mode http
         bind 0.0.0.0:80
         timeout client 50s
-        redirect prefix https://#OVERVIEW_ENVIRONMENT_ADDRESS# code 301 
+        redirect scheme https code 301
```

Comment:
This isn't the same thing. It leads to double-redirect: http://overviewproject.org => https://overviewproject.org => https://www.overviewproject.org. Please undo this line.